web/src/components/schema/Object.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <div>
+    <template v-for="(val, key) in data">
+      <v-row
+        v-if="val || (!val && !omitEmpty)"
+        :key="key"
+      >
+        <strong class="mr-2">{{ objectKey(key) }}:</strong> {{ renderedValue(val) }}
+      </v-row>
+    </template>
+  </div>
+</template>
+
+<script lang="ts">
+import { computed, defineComponent, PropType } from '@vue/composition-api';
+import type { JSONSchema7 } from 'json-schema';
+import type { JSONSchema7WithSubSchema } from '@/utils/schema/types';
+
+export default defineComponent({
+  name: 'Object',
+  props: {
+    data: {
+      // The data at the matching level of schema
+      type: Object as PropType<Record<string, any>>,
+      required: true,
+    },
+    schema: {
+      // The data at the matching level of schema
+      type: Object as PropType<JSONSchema7>,
+      required: false,
+      default: null,
+    },
+    options: {
+      type: Object as PropType<Record<string, any>>,
```

Comment:
`options` is intended to be a way to pass settings to the rendering sub components (`Object.vue` being one of those sub components) to modify behavior. Since the actual component being rendered is dynamic, the `options` prop is used to be agnostic to the component, while still allowing for some functionality. However I think you're right, these could probably be typed, so I can look into adding that. If it seems like it'll be too much effort right now, I can file an issue.

Reply:
14980da